vendor/github.com/elastic/gosigar/sigar_linux_common.go

```diff
@@ -282,6 +282,61 @@ func (self *ProcTime) Get(pid int) error {
 	return nil
 }
 
+func (self *ProcIO) Get(pid int) error {
+	// read proc io statistics from /proc/<pid>/io
```

Comment:
This might be a little simpler using `fmt.Scanf`. For example: https://github.com/elastic/procfs/blob/master/proc_io.go#L44-L52